.travis.yml

```diff
@@ -1,22 +1,41 @@
+sudo: false
 language: python
 python:
   - "2.7"
 # command to install wxPython using apt
-before_install:
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq python-wxgtk2.8
+# before_install:
+#   - sudo apt-get update -qq
+#   - sudo apt-get install -qq python-wxgtk2.8
 # command to install other dependencies
+
+addons:
+  apt:
+    packages:
+      # for wxPython:
+      - python-wxgtk2.8
+      - python-wxtools
+      - wx2.8-doc
+      - wx2.8-examples
+      - wx2.8-headers
+      - wx2.8-i18n
```

Comment:
Hi ethan,

Thanks for deleting of the commented lines.
For now we can also delete wx and may be, include it when we get to
improving testing for app modules to improve the coverage.

Regards

Henry S.

On Mon, Dec 7, 2015 at 4:46 PM, Ethan White <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/weecology/retriever/pull/367#discussion_r46882443>:
>
> >  # command to install other dependencies
> > +
> > +addons:
> > +  apt:
> > +    packages:
> > +      # for wxPython:
> > +      - python-wxgtk2.8
> > +      - python-wxtools
> > +      - wx2.8-doc
> > +      - wx2.8-examples
> > +      - wx2.8-headers
> > +      - wx2.8-i18n
>
> Do we need to install wx for testing purposes now that the retriever only
> loads it for the GUI following c73c39c
> <https://github.com/weecology/retriever/commit/c73c39c88dc5d873da7fa063ad11137d92d8a0b3>?
> I suspect we can remove the wx imports entirely.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/weecology/retriever/pull/367/files#r46882443>.
>